proto/cosmos/streaming/abci/v1/grpc.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package cosmos.streaming.abci.v1;
```

Comment:
why is this part of abci? abci should be a minimal layer between the app between modules and tendermint. Im leaning towards not adding more things to it like this

Reply:
This is under streaming. I called it abci because its streaming abci events. Where should it go?

Reply:
why was this moved out of store? this was moved into store to house all things storage within store. can you please move it back or provide reasoning for this move

Reply:
I'm confused. Are you mistaking it  for `proto/comsos/base/store/v1beta1/listening.proto`? This is a brand new grpc service and was never in store. If the above is not correct, please advice on what the package name and location of the proto should be.

Reply:
moved package to `store`.